notebooks/README.md

```diff
@@ -39,11 +39,11 @@ conda install ipykernel
 
 # for stable rapids version
 conda install -c rapidsai -c numba -c conda-forge -c nvidia \
-    cuxfilter=23.02 python=3.11 cudatoolkit=11.8
+    cuxfilter python=3.12 cudatoolkit=11.8
```

Comment:
The Python version change pulled this into the diff, which made me realize the outdated `cuxfilter=23.02` here. I definitely think that should be removed.

Should the `cudatoolkit` also be removed? I'd expect its dependencies like `cuspatial` and `cudf` to manage their own runtime dependencies on things from the CTK, and that separately installing `cudatoolkit` shouldn't be required.

Reply:
We should:
- Drop `-c numba` from the line above
- Replace this with something like what we have in cudf:
```
conda install -c rapidsai -c conda-forge -c nvidia \
    cuxfilter=24.10 python=3.12 cuda-version=12.5
```

Make sure `update-version.sh` has a replacement command for versions in `notebooks/README.md`.

Reply:
Ok great, will push that shortly.

Looks like that's what this project does already do in its main README, so it's just this notebook README that needs to be updated.

https://github.com/rapidsai/cuxfilter/blob/ca63e9d5fdfced7f993f18989ae42409185cff43/README.md#L156-L178

Reply:
Maybe we can reduce the content and only list it one place?

Reply:
yeah even better

Reply:
Just pushed a commit making these docs in `notebooks/README.md` point back to the installation steps in the top-level `README.md`.